src/input.h

```diff
@@ -627,6 +627,24 @@ class input_context
          * (x, y, width, height) would describe an area on the visible window that, if clicked, triggers the action.
          */
 
+        /**
+         * Used to implement mouse "edge scrolling". Returns a
+         * tripoint which is a vector of the resulting "move", i.e.
+         * (0, 0, 0) if the mouse is not at the edge of the screen,
+         * otherwise some (x, y, 0) depending on which edges are hit.
+         *
+         * @param speed The number of steps to scroll.
+         */
+        tripoint mouse_edge_scrolling( const int speed );
```

Comment:
`input` isn't a very good scope for these functions: despite their "general-purpose" names, they relate specifically to window management. The only thing they use from the `input` class itself is `get_raw_input`, which is it's main responsibility and therefore `public`.

Reply:
Ack, so I guess I should just revert commit 71e4a79005480f850fd4deb35ec14b18f40fe070 and let these functions live in `game`. Or do you have a better suggestion?

Reply:
Reverted.